features/partnerhistorie.feature

```diff
@@ -15,7 +15,7 @@ Functionaliteit: Tonen van Partnerhistorie
   Als de persoon tijdens (een deel van) de periode een huwelijk sloot of partnerschap aanging die aan het einde de gevraagde periode nog niet ontbonden is, dan wordt dat huwelijk of partnerschap opgenomen in het antwoord.
   Als de persoon tijdens de periode een huwelijk sloot of partnerschap aanging die voor het einde van de gevraagde periode werd ontbonden, dan wordt dat huwelijk of partnerschap opgenomen in het antwoord.
   Als partnerhistorie wordt gevraagd met peildatum gelijk aan de Datum ontbinding huwelijk/geregistreerd partnerschap van een huwelijk/partnerschap, dan wordt dat huwelijk of partnerschap niet opgenomen in het antwoord.
-  Als partnerhistorie wordt gevraagd met datumvan gelijk aan de Datum ontbinding huwelijk/geregistreerd partnerschap van een huwelijk/partnerschap, dan wordt dat huwelijk of partnerschap niet opgenomen in het antwoord.
+  Als partnerhistorie wordt gevraagd met datumVan gelijk aan de Datum ontbinding huwelijk/geregistreerd partnerschap van een huwelijk/partnerschap, dan wordt dat huwelijk of partnerschap niet opgenomen in het antwoord.
```

Comment:
In regel 7 van dit document staat

'Bij het ophalen van ontbonden huwelijken en partnerschappen uit GBA-V wordt Datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) uit de eerste historische huwelijk/partnerschap (categorie 55) na het actuele huwelijk/partnerschap waarin Datum huwelijkssluiting/aangaan geregistreerd partnerschap een waarde heeft.'

Volgens mij ontbreekt aan het eind nog het woord 'opgenomen'.

Overigens is de zin niet erg leesbaar.

Reply:
Aangepast. Met cursief geprobeerd de zin wat leesbaarder te maken.